apps/mobile/src/components/Feed/CommentsBottomSheet/CommentsBottomSheetLine.tsx

```diff
@@ -99,21 +125,47 @@ export function CommentsBottomSheetLine({
           </GalleryTouchableOpacity>
         </View>
       )}
-      <View className="flex flex-col grow-0">
-        <View className="flex flex-row space-x-1">
-          <Typography className="text-sm leading-4" font={{ family: 'ABCDiatype', weight: 'Bold' }}>
-            {comment.commenter?.username}
-          </Typography>
-          <Typography
-            className="text-xxs text-metal leading-4"
-            font={{ family: 'ABCDiatype', weight: 'Regular' }}
+      <View className="flex flex-col w-full flex-1">
+        <View className="flex-row justify-between">
+          <View>
+            <View className="flex-row space-x-1">
+              <Typography
+                className="text-sm leading-4"
+                font={{ family: 'ABCDiatype', weight: 'Bold' }}
+              >
+                {comment.commenter?.username}
+              </Typography>
+              <Typography
+                className="text-xxs text-metal leading-4"
+                font={{ family: 'ABCDiatype', weight: 'Regular' }}
+              >
+                {timeAgo}
+              </Typography>
+            </View>
+            <ProcessedText text={comment.comment} mentionsRef={nonNullMentions} />
+          </View>
+          <GalleryTouchableOpacity
+            className="flex-row items-center gap-0.5"
+            onPress={handleAdmirePress}
+            eventElementId="Press Admire Comment"
+            eventName="Press Admire Comment"
+            eventContext={contexts.Posts}
           >
-            {timeAgo}
-          </Typography>
+            {totalAdmires > 0 && (
+              <Typography
+                className={clsx('text-xs', {
+                  'text-activeBlue dark:text-darkModeBlue': hasViewerAdmiredComment,
+                  'text-shadow dark:text-metal': !hasViewerAdmiredComment,
+                })}
+                font={{ family: 'ABCDiatype', weight: 'Bold' }}
+              >
+                {totalAdmires}
+              </Typography>
+            )}
+            <AdmireIcon variant="secondary" height={16} active={hasViewerAdmiredComment} />
```

Comment:
looks like there's a small alignment issue when the comment is long
![CleanShot 2023-12-14 at 21 12 08@2x](https://github.com/gallery-so/gallery/assets/80802871/ca4819d4-0aed-4006-911b-7d77c78382cf)

Reply:
Nice catch! Fixed it 🫡

![CleanShot 2023-12-14 at 22 03 01@2x](https://github.com/gallery-so/gallery/assets/4480258/19baf7ad-48d9-4884-96c1-d43ffae47a7d)

Reply:
🥳🥳